src/main/java/com/angelbroking/smartapi/SmartConnect.java

```diff
@@ -804,6 +804,22 @@ public JSONObject logout() {
 			return null;
 		}
 	}
-    
+
+	/**
+	 * Get Individual Order Details
+	 *
+	 * @return JSONObject which contains order details from Smart API
+	 *
+	 */
+	public JSONObject getIndividualOrderDetails(String orderId) {
+		try {
+			String url = routes.get("api.individual.order").concat(orderId);
+			return smartAPIRequestHandler.getRequest(this.apiKey, url, accessToken);
+		} catch (SmartAPIException e) {
+            throw new RuntimeException(e);

```

Comment:
why catch (SmartAPIException e) {
            throw new RuntimeException(e);?

Reply:
The exceptions now handle correctly , please checl